repository_service_tuf_api/bootstrap.py

```diff
@@ -42,55 +43,50 @@ class Settings(BaseModel):
 
 
 class BootstrapPayload(BaseModel):
+    model_config = ConfigDict(json_schema_extra={"example": payload_example})
     settings: Settings
     metadata: Dict[Literal[Roles.ROOT.value], TUFMetadata]
-    timeout: Optional[int] = 300
-
-    class Config:
-        with open("tests/data_examples/bootstrap/payload.json") as f:
-            content = f.read()
-        example = json.loads(content)
-        schema_extra = {"example": example}
+    timeout: int | None = Field(default=300, description="Timeout in seconds")
 
 
 class PostData(BaseModel):
-    task_id: Optional[str]
+    task_id: str | None = None
     last_update: datetime
 
 
 class BootstrapPostResponse(BaseModel):
-    data: Optional[PostData]
-    message: str
-
-    class Config:
-        example = {
-            "data": {
-                "task_id": "7a634b556f784ae88785d36425f9a218",
-                "last_update": "2022-12-01T12:10:00.578086",
-            },
-            "message": "Bootstrap accepted.",
+    model_config = ConfigDict(
+        json_schema_extra={
+            "example": {
+                "data": {
+                    "task_id": "7a634b556f784ae88785d36425f9a218",
+                    "last_update": "2022-12-01T12:10:00.578086",
+                },
+                "message": "Bootstrap accepted.",
+            }
         }
-
-        schema_extra = {"example": example}
+    )
+    data: PostData | None = None
+    message: str
 
 
 class GetData(BaseModel):
     bootstrap: bool
-    state: Optional[str]
-    id: Optional[str]
+    state: str | None = None
```

Comment:
Why change type hint here and elsewhere?

Reply:
I thought it could be good to follow the FastAPI d[ocumentation for Python 3.10+](https://fastapi.tiangolo.com/tutorial/body/#__tabbed_1_1) as we support Python >=3.10